src/orquestra/sdk/exceptions.py

```diff
@@ -156,6 +156,12 @@ class InvalidWorkflowRunLogsError(BaseRuntimeError):
     pass
 
 
+class TaskRunLogsNotFound(NotFoundError):
+    """Raised when a task run logs cannot be found, or the ID is invalid"""
+
```

Comment:
**non-blocking,suggestion**: WDYT about adding `wf_run_id` and `task_inv_id` as fields here?

Reply:
will do